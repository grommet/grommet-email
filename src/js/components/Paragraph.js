// (C) Copyright 2017 Hewlett Packard Enterprise Development LP

import React, { PropTypes } from 'react';
import Style, { unit, halfUnit } from '../utils/Style';

const Paragraph = (props) => {
  const { children } = props;
  const style = {
    color: Style.color,
    margin: `${unit}px ${halfUnit}px`,
  };
  return (
    <p style={style}>
      {children}
    </p>
  );
};

Paragraph.propTypes = {
  children: PropTypes.any,
};

export default Paragraph;
