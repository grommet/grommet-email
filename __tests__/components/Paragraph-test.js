// (C) Copyright 2017 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Paragraph from '../../src/js/components/Paragraph';

describe('Paragraph', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Paragraph>Testing</Paragraph>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
