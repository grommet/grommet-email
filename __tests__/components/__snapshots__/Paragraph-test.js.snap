// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph has correct default options 1`] = `
<p
  style={
    Object {
      "color": "#666",
      "margin": "24px 12px",
    }
  }
>
  Testing
</p>
`;
